### PR TITLE
Remove the `dbg_func` and numParams options from function creation

### LINKED ISSUE #139: Cleanup custom function
> Remove the dbg_func and numParams options from Database.create_function. (use func.length instead of numParams). And adding and `@example` to the doc comment would be great too.

Assign to @dinedal

### PR DESCRIPTION
**Pull request by dinedal**
and adds better docs for function creation.

Closes #139

### CODE REVIEW

File: js/sql-debug.js [hunk 1]
```diff
@@ -275353,20 +275353,18 @@ Database = (function() {
 
 
   /* Register a custom function with SQLite
+  @example Register a simple function
+      db.create_function("addOne", (x) -> x + 1)
```
Comment (lovasoa) [inline js/sql-debug.js]: We usually keep our examples in traditional javascript. So 
```js
db.create_function("addOne", function(x) {return x+1;})
```
would be better.

**Comment by lovasoa** [comment]
Thank you, looks fine, I'm going to merge this :+1: 
But first can you add yourself to [AUTHORS](https://github.com/kripken/sql.js/blob/master/AUTHORS) and update [use instructions in the README](https://github.com/kripken/sql.js/blame/master/README.md#L54) ?

**Comment by lovasoa** [comment]
Thank you again :100:

**Comment by lovasoa** [comment]
I regenerated the documentation from your code: http://kripken.github.io/sql.js/documentation/class/Database.html#create_function-dynamic

And I didn't see it, but you used a coffeescript-style comment (`#`) in the example (that is supposed to be javascript). I'm going to fix that.

**Comment by dinedal** [comment]
Good catch, I'll be more mindful of this in the future.

**Pull request merged**